missions/2PzD_Template.VR/customization/loadoutsVeh/US/vehLoadoutUS.sqf

```diff
@@ -0,0 +1,100 @@
+// add this this file to your vehGear.sqf inside the switch like this.
+// #include "vehLoadoutUS.sqf"
+// then add the appropriate calls to the vehicle inits
+// then delete any loadouts you aren't using to save space.
+
+//Loadout Calls
+/*
+All years
+    [this, US_Resupply] call Olsen_FW_FNC_VehicleGearScript;         US Resupply Vehicle
+    [this, US_Squad] call Olsen_FW_FNC_VehicleGearScript;            US Squad Vehicle
+    [this, US_Platoon] call Olsen_FW_FNC_VehicleGearScript;          US Platoon Vehicle
+Tank 1942
+    [this, US_Tank_Early] call Olsen_FW_FNC_VehicleGearScript;       US Tank Early
+Tank 1944
+    [this, US_Tank_Late] call Olsen_FW_FNC_VehicleGearScript;        US Tank Late
+*/
+
+//======================== Definitions ========================
+
+#define US_Infantry_Supplies \
+    [US_Mag_M1C, 6] call Olsen_FW_FNC_AddItemVehicle; \
+	[US_Mag_M1T_20, 6] call Olsen_FW_FNC_AddItemVehicle; \
+    [US_Mag_BAR_Mixed_Ball, 20] call Olsen_FW_FNC_AddItemVehicle; \
```

Comment:
Might bump this up to 25 even, seeing as the 20rnd magazine is slightly less than half the 50rnd German one.